Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -281,6 +282,29 @@ void PixelShaderManager::SetFogRangeAdjustChanged()
   s_bFogRangeAdjustChanged = true;
 }
 
+void PixelShaderManager::SetBlendMode()
+{
+  if (g_ActiveConfig.backend_info.bSupportsFramebufferFetch)
+  {
+    BlendingState state;
+    state.Generate(bpmem);
+    constants.blend_enable = state.blendenable;
+    constants.blend_dstalpha = state.dstalpha;
+    constants.blend_subtract = state.subtract;
+    constants.blend_subtractalpha = state.subtractAlpha;
+    constants.blend_dstfactor = state.dstfactor;
+    constants.blend_srcfactor = state.srcfactor;
+    constants.blend_dstfactoralpha = state.dstfactoralpha;
+    constants.blend_srcfactoralpha = state.srcfactoralpha;
+
+    dirty = true;
+  }
+  else
```

Comment:
This can just be omitted, or just be de-indented with:

```cpp
if (!g_ActiveConfig.backend_info.bSupportsFramebufferFetch)
  return;

...
```

no?